content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -668,27 +668,29 @@ consume.
 
 ## Job patterns
 
-The Job object can be used to support reliable parallel execution of Pods. The Job object is not
-designed to support closely-communicating parallel processes, as commonly found in scientific
-computing. It does support parallel processing of a set of independent but related *work items*.
-These might be emails to be sent, frames to be rendered, files to be transcoded, ranges of keys in a
-NoSQL database to scan, and so on.
+The Job object can be used to process a set of independent but related *work items*.
+These might be emails to be sent, frames to be rendered, files to be transcoded,
+ranges of keys in a NoSQL database to scan, and so on.
 
 In a complex system, there may be multiple different sets of work items. Here we are just
 considering one set of work items that the user wants to manage together &mdash; a *batch job*.
 
 There are several different patterns for parallel computation, each with strengths and weaknesses.
 The tradeoffs are:
 
-- One Job object for each work item, vs. a single Job object for all work items. The latter is
-  better for large numbers of work items. The former creates some overhead for the user and for the
-  system to manage large numbers of Job objects.
+- One Job object for each work item, vs. a single Job object for all work items. The former creates
+  some overhead for the user and for the system to manage large numbers of Job objects. The latter
+  is better for large numbers of work items. 
 - Number of pods created equals number of work items, vs. each Pod can process multiple work items.
   The former typically requires less modification to existing code and containers. The latter
   is better for large numbers of work items, for similar reasons to the previous bullet.
 - Several approaches use a work queue. This requires running a queue service,
   and modifications to the existing program or container to make it use the work queue.
   Other approaches are easier to adapt to an existing containerised application.
+- When the Job is accompanied with a
+  [headless Service](/docs/concepts/services-networking/service/#headless-services)
+  you can enable the processes within a Job to communicate, as commonly necessary
+  in scientific computing.
```

Comment:
Addressing @vsoch comment, maybe just drop the "as commonly necessary in scientific computing" phrase altogether?

Reply:
I rephrased to remove "scientific computing". WDYT?

Reply:
> That kind of approach would also mean it's vendor natural, b/c I assume based on what you wrote that you basically are running DIY k8s, right?

For our needs we are using hardware provided by a daemonset from AWS (elastic fiber adapter) and it's deployed fairly easily with eksctl. We are still investigating what the equivalent is on GKE. So it's vendor-specific (for the time being) but I wouldn't consider it DIY k8s.

What would be fantastic to do (if/when we are allowed) is to write some kind of post that describes our findings for how to run these latency sensitive workloads, maybe even on different clouds, when we figure it out here. I think there could be a way to do that without doing a direct comparison of performance details between vendors but generally referencing best practices!

Reply:
> I rephrased to remove "scientific computing". WDYT?

That would actually work - I might suggest saying one level up - that it enables the _pods_ within a Job to communicate. I wouldn't go into processes.